components/browser_kit.rst

```diff
@@ -279,6 +280,42 @@ also delete all the cookies::
     // reset the client (history and cookies are cleared too)
     $client->restart();
 
+.. _ component-browserkit-external-requests:
+
+Making External HTTP Requests
+-----------------------------
+
+So far, all the examples in this article have assumed that you are making
+internal requests to your own application. However, you can run the exact same
+examples when making HTTP requests to external web sites and applications.
+
+First, install and configure the :doc:`HttpClient component </components/http_client>`.
+Then, use the :class:`Symfony\\Component\\HttpClient\\BrowserClient` to create
+the client that will make the external HTTP requests::
+
+    use Symfony\Component\HttpClient\BrowserClient;
+    use Symfony\Component\HttpClient\HttpClient;
```

Comment:
What is BrowserClient?
I can't find the class in the http-client component.
Isn't it
```php
use Symfony\Component\BrowserKit\HttpBrowser;
...
$browser = new HttpBrowser();
```

Reply:
You are right! That's the class: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/BrowserKit/HttpBrowser.php

I've fixed the example. Thanks!